subjects/mobile-dev/stock-market/resources/mock-stock-data-server/Makefile

```diff
@@ -0,0 +1,18 @@
+IMAGE_NAME="stock-server"
+SERVER_PORT=5000
+PUBLIC_PORT=5000
```

Comment:
You may need to change the port number to 5001, since for mac users this port is used by default by `ControlCenter.app`
see https://apple.stackexchange.com/a/431164